File: spring_2021/coding_meetup_02/true_or_false_exam/exam.py
# @author Benjamin Alcocer

friendCorrect = int(input())
myAnswers = input().replace('\n', '')
friendAnswers = input().replace('\n', '')

same = 0
size = len(myAnswers)
for i in range(size):
    if myAnswers[i] == friendAnswers[i]:
        same += 1
print(min(friendCorrect, same) + min(size - same, size - friendCorrect))

File: 2021_spring/coding_meetup_02/true_or_false_exam/exam.cpp
/// @author Benjamin Alcocer

#include <iostream>
#include <string>

using namespace std;

int main()
{
    string mine, other;
    int ansAmount, same;
    
    cin >> ansAmount >> mine >> other;
    for (int i = 0; i < mine.size(); ++i)
    {
        if (mine[i] == other[i])
            ++same;
    }
    cout << min(ansAmount, same) + min(mine.size() - same, mine.size() - ansAmount);

    return 0;
}
